src/handbook/company/security/risk-management.md

```diff
@@ -0,0 +1,114 @@
+and/or processes.
+
+Risk assessments must be performed with any major change to FlowForge's business
+or technical operations and/or supporting infrastructure no less than once per year.
+
+Strategies shall be developed to mitigate or accept the risks identified in the
+risk assessment process.
+
+A [risk register](https://app.vanta.com/risk-management/risk-register) is maintained
+and monitored quarterly to assess compliance with the above policy, and document
+newly discovered or created risks.
+
+## Managing Risk
+
+Risks are assessed and ranked according to their impact and their likelihood of
+occurrence. A formal Risk Assessment, and network penetration tests, will be
+performed at least annually and shall take into consideration the results of any
+technical vulnerability management activities performed in accordance with the
+[Operations Security Policy](./operations-security.md).
+
+Each risk will be assessed as to its likelihood and impact. Both impact and
+likelihood are assessed on a scale of 1-5. Impact can range from 1 ("Very low impact")
+to 5 ("Very high impact") and likelihood can range from 1 ("Very unlikely") to 5 ("Very likely").
+
+The criteria for determining risk is the combined likelihood and impact of an
+event adversely affecting the confidentiality, availability, integrity, or
+privacy of organizational and customer information, personally identifiable
+information (PII), or business information systems.
+
+For all risk inputs such as risk assessments, vulnerability scans, penetration test,
+bug bounty programs, etc., FlowForge management shall reserve the right to modify
+risk rankings based on its assessment of the nature and criticality of the system
+processing, as well as the nature, criticality and exploitability (or other
+relevant factors and considerations) of the identified vulnerability.
+
+## Risk Response, Treatment, and Tracking
+
+Risk will be prioritized and maintained in a risk register where they will be
+prioritized and mapped using the approach contained in this policy.
+
+The following responses to risk should be employed:
+
+ - **Modify**: take actions or employ strategies to reduce the risk.
+ - **Accept**: accept and monitor the risk at the present time. This may be necessary for some risks that arise from external events.
+ - **Transfer**: pass the risk on to another party. For example contractual terms may be agreed to ensure that the risk is not borne by FlowForge or insurance may be appropriate for protection against financial loss.
+ - **Avoid**: cease the activity or to change it in such a way as to end the risk.
+
+Where FlowForge chooses a risk response other than "Accept" or "Avoid" it shall
+develop a Risk Treatment Plan
+
+
+### Risk Management Procedures
+
+The procedure for managing risk will meet the following criteria:
+
+ - FlowForge will maintain a [Risk Register]((https://app.vanta.com/risk-management/risk-register)) and Treatment Plan.
```

Comment:
```suggestion
 - FlowForge will maintain a [Risk Register](https://app.vanta.com/risk-management/risk-register) and Treatment Plan.
```

Reply:
Why not just a Google Sheet? We'd like everyone in the org to have access to it

Reply:
Vanta provide all the purpose-built tooling for this which also then feeds into our compliance dashboard. Didn't want to reinvent the wheel when they have it all there ready for us to use. Everyone will have access to it.

Reply:
@knolleary That introduces another source of data, and I for one never use Vanta. Nor do I think we should depend too much on them other than obtaining a SOC2. Further; we're paying Vanta 18k/year for it, we should be ready to not use Vanta and go to a competitor if they will cost say $10k

Reply:
The alternative view is we have paid them for their tooling that can help expedite the SOC2 process as it is proven audit-ready and fully documented.

Going our own way will require working out all of the process around this for ourselves. I'm trying to minimise our workload here. But if that is the approach you'd prefer, we can make it so.